standards/3-commercial/com/payment.md

```diff
@@ -68,16 +71,21 @@ golem.com.payment.platform.zksync-mainnet-glm.address = "0xdeadbeef"
 Demand 1
 
 Constraints:
-()
+(golem.com.payment.protocol.version>1)
```

Comment:
And I forgot to mention previously that the property should be added to `cheat_sheet.md` as well